### ISSUE TITLE
Accessibility - Focus border around radio buttons does not meet contrast requirement

### ISSUE THREAD

**Issue by Revanuru** [localgovdrupal/localgov_base#474]
My actions were as follows:
1. Launch the below URL
2. Tab around the page until you reach the radio buttons
3. A yellow border on a white background will be displayed around the radio buttons

URL: https://beta.intranet.essex.gov.uk/form/feedback-form?reference=99#no-back
Compliance with WCAG 1.4.11 https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html:

Actual result: Focus border does not meet the minimum contrast requirement of 3:1
Expected result: There is decent contrast for graphics (that don't have text) and inputs/controls/focus styles.

Attention: LGD styling issue, please can someone look at this one, thanks.

**Comment by markconroy** [comment]
Thanks @Revanuru 

This is a very interesting one, as we are following the Gov UK design system for our form items. Here's their radios: https://govuk-elements.herokuapp.com/form-elements/#form-radio-buttons

Here's their default state:

![Screenshot 2023-07-21 at 13 12 39](https://github.com/localgovdrupal/localgov_base/assets/2183332/2c43a5c6-44e7-4763-8a33-bf302013247d)

When you tab to a radio item, here's what you see (I agree the contrast should be more, or a thicker black outline around the circle

![Screenshot 2023-07-21 at 13 12 47](https://github.com/localgovdrupal/localgov_base/assets/2183332/16b0c5e3-73dd-4bdb-bab6-d1fc0a75438d)

Then when you select something, here's what you see (which looks perfectly contrasting)
![Screenshot 2023-07-21 at 13 12 56](https://github.com/localgovdrupal/localgov_base/assets/2183332/9df7df37-a6c2-43f7-aad0-6a391ba9f5fe)

@msayoung Any thoughts on what we should do with this?

**Comment by msayoung** [comment]
Great spot @Revanuru ,

For reference we have two new WCAG criteria on the horizon, which cover this exact question.

- https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum
- https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-enhanced

It looks like gov.uk have updated their design system too now, and that looks like a good pattern for us to follow @markconroy 

https://design-system.service.gov.uk/components/radios/

![image](https://github.com/localgovdrupal/localgov_base/assets/3852805/07895cb5-82a1-4cf1-914c-9102293e2d04)

The focused radio input in this example has a thicker border ( from 2px to 4px ) and still keeps the yellow outline. 

I think this applies to all our form inputs, doesn't it?

**Comment by markconroy** [comment]
@Revanuru can you test PR #475 to see if it fixes this please?

**Comment by Revanuru** [comment]
@markconroy thanks so much for your reply, would you please let me know where should I test this? Sorry this is the first time I'm working with LGD. Appreciate your help, Thanks Sucharitha

**Comment by markconroy** [comment]
You can test it locally on a dev site for LGD. 

1. Install LGD
2. Remove the localgov_base theme from the code
3. Use git to download the latest version of localgov_base
4. Check out the branch in the #475 PR
5. Clear the cache and you should see the new change.

**Comment by markconroy** [comment]
hmmm ... just realised, I am presuming you are a developer. If not, let me know and I can get someone else to test this for us.

**Comment by Revanuru** [comment]
Thanks @markconroy  - unfortunately I'm not & we don't have dev environment, we depend on third party (Nomensa) for development, they do the dev & deployments for us!

**Comment by Revanuru** [comment]
Appreciate if you can get that tested and let me know the outcome! Thank you

**Comment by msayoung** [comment]
@markconroy - I'll give this a test for you.

**Comment by msayoung** [comment]
Gorgeous @markconroy 

![image](https://github.com/localgovdrupal/localgov_base/assets/3852805/d961b895-6a2e-4e76-99e5-6d50eeaa7646)
